twistr/events.go

```diff
@@ -597,6 +727,22 @@ func PlayMissileEnvy(s *State, player Aff) {
 	   immediately. If it contains an opponent’s Event, use the Operations value
 	   (no Event). The opponent must use this card for Operations during their next
 	   action round. */
+	s.TurnEvents[MissileEnvy] = player
```

Comment:
missileenvy is not a turn event; it is an event.

"The opponent must use this card for Operations during their next action round." US player can play this for AR7, forcing USSR to play missile envy for ops in her next AR1.